content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+gl-sbp-glhcnimblestor             csi.hpe.com                    Delete          Immediate              true                   69d
+hpe-hdd-storage                   kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  69d
+hpe-nvme-storage                  kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  69d
+hpe-ssd-storage                   kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  69d
+
+$ kubectl get all -n hpe-storage
+NAME                                       READY   STATUS    RESTARTS   AGE
+pod/hpe-csi-controller-7c6f876494-vrd49    9/9     Running   0          69d
+pod/hpe-csi-node-cpmkg                     2/2     Running   0          69d
+pod/hpe-csi-node-m2f75                     2/2     Running   0          69d
+pod/hpe-csi-node-m9mj9                     2/2     Running   0          69d
+pod/nimble-csp-db7c7bb65-c5wrk             1/1     Running   0          69d
+pod/primera3par-csp-6f999b8d76-wtd4n       1/1     Running   0          69d
+pod/snapshot-controller-0                  1/1     Running   0          69d
+pod/snapshot-controller-64b98b668f-rdfwr   1/1     Running   0          32d
+pod/snapshot-controller-64b98b668f-tdzz9   1/1     Running   0          32d
+
+NAME                          TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+service/alletra6000-csp-svc   ClusterIP   10.109.228.208   <none>        8080/TCP   69d
+service/alletra9000-csp-svc   ClusterIP   10.107.227.232   <none>        8080/TCP   69d
+service/nimble-csp-svc        ClusterIP   10.102.45.17     <none>        8080/TCP   69d
+service/primera3par-csp-svc   ClusterIP   10.104.57.79     <none>        8080/TCP   69d
+
+NAME                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/hpe-csi-node   3         3         3       3            3           <none>          69d
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpe-csi-controller    1/1     1            1           69d
+deployment.apps/nimble-csp            1/1     1            1           69d
+deployment.apps/primera3par-csp       1/1     1            1           69d
+deployment.apps/snapshot-controller   2/2     2            2           32d
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/hpe-csi-controller-7c6f876494    1         1         1       69d
+replicaset.apps/nimble-csp-db7c7bb65             1         1         1       69d
+replicaset.apps/primera3par-csp-6f999b8d76       1         1         1       69d
+replicaset.apps/snapshot-controller-64b98b668f   2         2         2       32d
+
+NAME                                   READY   AGE
+statefulset.apps/snapshot-controller   1/1     69d
+```
+
+### Step-5: Create a namespace on 'hpe' Kubernetes cluster for MongoDB Deployment
```

Comment:
Again, with whatever the next step is, give a little explanation just before you put in all that code. Also "Deployment" should be "deployment"